cms_product_form/controllers/main.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2018 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import http
+from odoo.addons.cms_form.controllers.main import FormControllerMixin
+from odoo.addons.cms_form.controllers.main import SearchFormControllerMixin
+
+
+class ProductFormController(http.Controller, FormControllerMixin):
+    """Partner form controller."""
+
+    @http.route([
+        '/product/add',
```

Comment:
you should override the cms_add/edit/search_url on template model. See https://github.com/OCA/website-cms/blob/11.0/cms_form/models/website_mixin.py#L11.

NOTE: in https://github.com/OCA/website-cms/pull/45 I'm moving those basic URLs and I'm renaming `add` to `create`. 
So better if: 
1. you review that PR (tnx!)
2. use `cms_create_url` instead of the soon outdated `add` :wink: